addons/mail/controllers/discuss/channel.py

```diff
@@ -3,39 +3,18 @@
 from werkzeug.exceptions import NotFound
 
 from odoo import http
-from odoo.exceptions import UserError
 from odoo.http import request
-from odoo.tools import consteq
 from odoo.addons.mail.models.discuss.mail_guest import add_guest_to_context
 
 
 class ChannelController(http.Controller):
     @http.route("/discuss/channel/members", methods=["POST"], type="json", auth="public")
     @add_guest_to_context
     def discuss_channel_members(self, channel_id, known_member_ids):
-        channel_member = request.env["discuss.channel.member"]._get_as_sudo_from_context_or_raise(channel_id=channel_id)
-        return channel_member.channel_id.sudo().load_more_members(known_member_ids)
-
-    @http.route("/discuss/channel/add_guest_as_member", methods=["POST"], type="json", auth="public")
-    @add_guest_to_context
-    def discuss_channel_add_guest_as_member(self, channel_id, channel_uuid):
```

Comment:
Note: I'm dropping this feature completely.
The only reason we delayed adding the member was to not have a stupid "joined" message that always had "Guest" as name, before the name was selected the first time.
Now the name in the notification automatically updates on rename.